plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -97,7 +97,7 @@ public void testShowCreateTable()
                         "   comment varchar\n" +
                         "\\)\n" +
                         "WITH \\(\n" +
-                        "   format = '" + format.name() + "',\n" +
```

Comment:
why this change?

Reply:
I got confused by some implementations of BaseIcebergConnectorSmokeTest not setting `"iceberg.file-format", format.name()`. I've removed this change and fixed the tests to set `"iceberg.file-format", format.name()`